open_event/helpers/oauth.py

```diff
@@ -0,0 +1,21 @@
+class OAuth(object):
+	"""Google Credentials"""
+
+	CLIENT_ID='449612261522-1eg34prt23l0454et59qgqno3rjd8muq.apps.googleusercontent.com'
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Found indentation with tabs instead of spaces](https://www.codacy.com/app/triplez/open-event-orga-server/file/2562226813/issues/source?bid=3233864&fileBranchId=3293867#l4)